tests/planning/methods/PlanToEndEffectorOffset.py

```diff
@@ -100,7 +100,7 @@ def test_PlanToEndEffectorOffset_GoalInCollision_Throws(self):
             self.planner.PlanToEndEffectorOffset(
                 self.robot, direction=self.direction, distance=self.distance)
 
-    def test_PlanToEndEffectorOffset_GoalInCollision_Throws(self):
+    def test_PlanToEndEffectorOffset_GoalInCollision_Throws2(self):
```

Comment:
There's another test with the same name, right above this. I'll change both of their names to be something more distinguishable.